test/ZendViewRendererTest.php

```diff
@@ -246,6 +246,80 @@ public function testTemplateDefaultParameterIsNotAvailableInLayout()
         $this->assertContains($expected, $result, sprintf('Received %s', $result));
     }
 
+    public function testTemplateDefaultParameterIsAvailableInLayout()
```

Comment:
I would suggest to rename this to testLayoutTemplateDefaultParameterIsAvailableInLayout as well